static/templates/stream_settings/stream_types.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="grey-box new-style">
+<div class="grey-box">
```

Comment:
`new-style` means something specific -- it refers to the last redesign we did to the settings buttons UI.  I think our goal should probably be to just have it be the default style for the application.

I think the Micromodal migration is going to end up removing a very large portion of explicit references to `new-style`, so we might be in a position to try to clean that up in a month or two.  May be worth a #frontend thread.

Reply:
But I also think this suggests that it's not super constructive to rework elements to assume that the `new-style` styling isn't present, since our plan is to make that replace Boostrap as the default button (etc.) styles.

Reply:
Given that it's literally one CSS block that you're adding in this commit, it's probably fine to keep things as-is if including `new-style` makes life more difficult.  

Actually I'm a bit puzzled; `static/templates/stream_settings/stream_settings_overlay.hbs` already has the `new-style` class.  Is the situation that the model doesn't inherit it, but the "stream creation form" overlay does?